GraphQL API docs (#19)
mirumee/saleor-docs: docs/api/playground.md

title: How to Access the Playground
---

> GraphQL Playground is external interactive editor for your GraphQL queries. It is based on the GraphiQL and accessible through the web browser. 

Reviewer: ```suggestion
GraphQL Playground is an external interactive editor for your GraphQL queries. It is based on the GraphiQL and accessible through the web browser. 
```

> With the `DEBUG=True` setting enabled, Saleor exposes an interactive GraphQL editor under `/graphql`, which allows accessing the API from your browser. To view and try our API out, visit the [Playground](https://demo.getsaleor.com/graphql/).

Reviewer: ```suggestion
With the `DEBUG=True` setting enabled, Saleor exposes an interactive GraphQL editor under `/graphql`, allowing to access the API from your browser. To view and try out our API, visit the [Playground](https://demo.getsaleor.com/graphql/).
```

Using Playground is easy and intuitive with its sidebar navigation containing two tab (_Docs_ and _Schema_), listing all operations available in Saleor API. The Playground allows you to quickly familiarize yourself with API, perform example operations and send your first queries.

> SCREENSHOT OF PLAYGROUND SALEOR WELCOME MESSAGE

Reviewer: ```suggestion
<!-- FIXME: SCREENSHOT OF PLAYGROUND SALEOR WELCOME MESSAGE -->
```
